study.md

```diff
@@ -85,7 +91,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+p "My name is #{:given_name} #{:family_name} and I'm an #{:occupation} that enjoys #{:hobbies[1]}."
```

Comment:
This will output `"My name is given_name family_name and I'm an occupation that enjoys o."` because it's just printing out the symbols.